README.md

```diff
@@ -1,14 +1,12 @@
-# shinyapps.io
-Welcome to [shinyapps.io](https://www.shinyapps.io)!
+# NOTICE
 
-## Getting Started
+The `shinyapps` R package is deprecated and has been replaced by the [rsconnect](https://github.com/rstudio/rsconnect) package.
 
-To get started using shinyapps.io checkout the [Getting Started Guide](http://shiny.rstudio.com/articles/shinyapps.html).
+If you're looking for support for shinyapps.io, checkout the [shinyapps.io User Guide](http://docs.rstudio.com/shinyapps.io/),
+or feel free to post a message to the [shinyapps-users](https://groups.google.com/forum/#!forum/shinyapps-users) discussion group.
 
-## Getting Help
-
-Need help? Checkout the [shinyapps.io User Guide](http://docs.rstudio.com/shinyapps.io/).
+Customers with paid subscriptions can open a ticket with [RStudio Support](https://rstudioide.zendesk.com).
```

Comment:
Should refer to https://support.rstudio.com (which redirects, but protects against switching away from Zendesk, which not likely, but still...)